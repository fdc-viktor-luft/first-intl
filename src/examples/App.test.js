// @flow

import React from 'react';
import { shallow } from 'enzyme';
import { App } from './App';

describe('App', () => {
    it('renders the App without translations, but all necessary information', () => {
        // see the "__snapshots__" directory within this directory to the see the rendered output
        expect(shallow(<App />)).toMatchSnapshot();
    });
});
