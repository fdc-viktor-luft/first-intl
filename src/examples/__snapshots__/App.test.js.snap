// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the App without translations, but all necessary information 1`] = `
<div>
  <header>
    <img
      alt="__({ id: 'header.logo.alt' })"
      src="/path/to/img"
    />
  </header>
  <main>
    <button>
      __({ id: 'account.remove' })
    </button>
    <p>
      __({ id: 'account.remove.info', values: {"email":"my.example@mail.com"} })
    </p>
    __({ id: 'account.tac.text', values: {"tac":{"type":"a","key":null,"ref":null,"props":{"href":"/path/to/tac","children":"__({ id: 'account.tac' })"},"_owner":null,"_store":{}},"info":"__({ id: 'account.tac.info' })","br":{"type":"br","key":null,"ref":null,"props":{},"_owner":null,"_store":{}}} })
  </main>
  <footer>
    <a
      href="/path/to/home"
    >
      __({ id: 'footer.navigation.home' })
    </a>
  </footer>
</div>
`;
